wxt.config.ts

```diff
@@ -20,6 +20,7 @@ export default defineConfig({
   manifest: ({ browser }) => {
     const manifest: UserManifest = {
       permissions: ["storage"],
+      default_locale: "en",
```

Comment:
Remember to add this, otherwise types for `i18n.t`/`i18n.tp` won't be generated.